src/sentry/static/sentry/app/views/performance/transactionDetails/quickTrace.tsx

```diff
@@ -34,15 +42,22 @@ export default function QuickTrace({event, organization, location}: Props) {
         ) : error || trace === null ? (
           '\u2014'
         ) : (
-          <QuickTraceLite event={event} trace={trace} />
+          <QuickTraceLite
+            event={event}
+            trace={trace}
+            location={location}
+            organization={organization}
+          />
         );
 
         return (
           <MetaData
             headingText={t('Quick Trace')}
-            tooltipText={t('The unique ID assigned to this transaction.')}
+            tooltipText={t('A minified version of the full trace.')}
             bodyText={body}
-            subtext={t('Trace ID: %s', getShortEventId(traceId))}
+            subtext={
+              trace === null ? '\u2014' : t('Trace ID: %s', getShortEventId(traceId!))
```

Comment:
Can we make this a link to discover for now too?

Reply:
That's a good idea. Will add that in a follow-up.